### DISCUSSION TITLE
Error to install cplex in or_tools using c++: error C2065: 'off64_t': identificador nao declarado

### DISCUSSION THREAD

**Discussion by mayracssantos** [google/or-tools#3653] [Q&A]
I'm trying to instal the Cplex solver in Or-Tools using c++, on : 
Windows 10
cmake 3.25.2
However, this is the error:
[errorortools.txt](https://github.com/google/or-tools/files/10519380/errorortools.txt)

and this is the file CMakeError:
[CMakeError.log](https://github.com/google/or-tools/files/10519399/CMakeError.log)

**Comment by Mizux** [comment]
seems to comes from zlib and our patch in patches/zlib.patch which has been generated from a linux machine using the autotools to get the `zconf.h` file
